### PR TITLE
fix #30

### LINKED ISSUE #30: Property names with special characters (ä,ö,ü) are converted to unicode strings
Hello,

some of the columns in our database contains special characters (ä,ö,ü).
The characters are converted to `&#228;`, `&#246;` or `&#252;` unicode strings.

To solve the issue for me, I have created a hbs-helper, which simple writes the property-name with the WriteSafeString-Method of the TextWriter.

Helper registration:
```csharp
    public void ConfigureDesignTimeServices(IServiceCollection services)
    {
        var options = ReverseEngineerOptions.DbContextAndEntities;
        services.AddHandlebarsScaffolding(options);
        
        Handlebars.RegisterHelper("f-pn", FormatPropertyName);
    }

    void FormatPropertyName(TextWriter writer, object context, object[] args)
    {
        writer.WriteSafeString(args[0].ToString());
    }
```
Usage in templates:
`{{f-pn property-name}}`

I think, the property-name template should use the WriteSafeString-Method.

### PR DESCRIPTION
**Pull request by Varorbc**
fix #30 @tonysneed 
image

### CODE REVIEW

File: sample/ScaffoldingSample.Api/ScaffoldingSample.Api.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.3" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.9" />
```
Comment (ErikEJ) [inline sample/ScaffoldingSample.Api/ScaffoldingSample.Api.csproj:9]: Is is really a good idea for a library to update this dependency to a very new version?

Comment (tonysneed) [reply sample/ScaffoldingSample.Api/ScaffoldingSample.Api.csproj:9]: Good question. I was thinking about upgrading the SqlServer dependency to 5.0.10, but it has 0 downloads -- LOL! What's the downside of upgrading to latest?

BTW, it looks like v6 RC1 just dropped a few hours ago. Time for me to start working on a version based on EF Core 6. :)

Comment (ErikEJ) [reply sample/ScaffoldingSample.Api/ScaffoldingSample.Api.csproj:9]: Downside is that your library forces an upgrade of the EF Core dependency on all your consumers, which may not suit them (for various reasons)

Comment (tonysneed) [reply sample/ScaffoldingSample.Api/ScaffoldingSample.Api.csproj:9]: Which version of SqlServer does EFC PT use?

Comment (ErikEJ) [reply sample/ScaffoldingSample.Api/ScaffoldingSample.Api.csproj:9]: I actually just upgraded to 5.0.9, but EFCPT is not a good example of a consumer, as I have all my EFCore dependencies isolated and distributed as a command line tool.

Comment (tonysneed) [reply sample/ScaffoldingSample.Api/ScaffoldingSample.Api.csproj:9]: Just noticed that this is in the sample proj. The lib does not have a dependency on `Microsoft.EntityFrameworkCore.SqlServer`, just `Abstractions` and `Design,` which **should** be backwards compatible.

Comment (ErikEJ) [reply sample/ScaffoldingSample.Api/ScaffoldingSample.Api.csproj:9]: I always stay at 3.1.0 / 5.0.0 etc with my EF Core releated libraries

Comment (tonysneed) [reply sample/ScaffoldingSample.Api/ScaffoldingSample.Api.csproj:9]: Thanks for the tip! I'll be sure to do that going forward.

**Pull request merged**